core-plugin/src/com/google/gct/idea/debugger/actions/CloudDebugHelpAction.java

```diff
@@ -0,0 +1,32 @@
+package com.google.gct.idea.debugger.actions;
+
+import com.intellij.CommonBundle;
+import com.intellij.ide.BrowserUtil;
+import com.intellij.openapi.actionSystem.AnAction;
+import com.intellij.openapi.actionSystem.AnActionEvent;
+import com.intellij.openapi.actionSystem.Presentation;
+import com.intellij.openapi.util.IconLoader;
+
+import javax.swing.Icon;
+
+/**
+ * Opens the Cloud Debugger help page.
+ */
+public class CloudDebugHelpAction extends AnAction {
+  private static final Icon icon = IconLoader.getIcon("/actions/help.png");
+  private String url;
+
+  public CloudDebugHelpAction(String url) {
+    this.url = url;
+  }
+
+  public void actionPerformed(AnActionEvent event) {
```

Comment:
Yes, but that doesn't test both scenarios when the URL is set in the parameters and when it isn't.

I should probably write a unit test for this class anyways.